### PR TITLE
fix : add instructions for resolving the support library not found error

### LINKED ISSUE #809: Failed to resolve issues while adding firebaseAuth library
* Android device: OnePlus3
  * Android OS version: 7.1.2
  * Google Play Services version: 11.0.2
  * Firebase/Play Services SDK version: 11.0.2
  * FirebaseUI version: 2.1.0
  
### Step 3: Describe the problem:

Upon adding the CardView, design and CustomTabs dependencies, the cradle sync fails with the following message,
screen shot 2017-07-19 at 2 23 11 pm

#### Steps to reproduce:

  1. Add the FirebaseUI Auth dependency.
  2. Add the dependencies required by it.
  3. Perform a gradle sync. 

  
#### Expected Results:

  The Sync should complete normally and the relevant dependencies should be added.

### PR DESCRIPTION
**Pull request by the-dagger**
Adds instruction into documentation to fix #809

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,34 @@ compile "com.android.support:appcompat-v7:$BAR"
 compile "com.android.support:palette-v7:$BAR"
 ```
 
+NOTE :
+The support libraries are now available through Google's Maven repository, so ensure that you have that added to your project's repositories.
+
+Open the `build.gradle` file for your project and modify it according to your gradle plugin version.
+
+If your gradle plugin is above 2.3.3
```
Comment (samtstern) [inline README.md]: Let's just drop this section.  Simpler to add the long form, which will work for everyone,

File: README.md [hunk 2]
```diff
@@ -136,6 +136,34 @@ compile "com.android.support:appcompat-v7:$BAR"
 compile "com.android.support:palette-v7:$BAR"
 ```
 
+NOTE :
+The support libraries are now available through Google's Maven repository, so ensure that you have that added to your project's repositories.
```
Comment (samtstern) [inline README.md]: Let's change this to mention the specific version when things changed.  Something like "as of version `25.4.0` the support libraries are available only through Google's Maven repository..."

Comment (samtstern) [inline README.md]: We should also link "Google's Maven repository" to here:
https://developer.android.com/studio/build/dependencies.html#google-maven

Comment (the-dagger) [inline README.md]: @samtstern Thanks for the review, made the changes 🙂

**Pull request merged**